templates/base/head.tmpl

```diff
@@ -72,6 +72,12 @@
 {{end}}
 </head>
 <body>
+	<script>
+		document.body.style.display = 'none';
+		document.addEventListener('DOMContentLoaded', function() {
+			document.body.removeAttribute('style');
```

Comment:
I went with it because if one just resets the display property, a ugly `style=""` property gets left over. But I agree, it can be problematic. I'll check for better alternatives.